package.json

```diff
@@ -2,13 +2,17 @@
   "name": "uw-frame",
   "version": "2.9.0",
   "description": "MyUW's frame project for creating angular based microservices.",
+  "config" : {
+    "javaversion" : "1"
```

Comment:
What's `javaversion`? Under what circumstances ought it to be incremented? How would I know that?